newrelic/resource_newrelic_alert_policy_channel_test.go

```diff
@@ -32,6 +33,39 @@ func TestAccNewRelicAlertPolicyChannel_Basic(t *testing.T) {
 				),
 			},
 			// Test: Import
+			{
+				ResourceName:     resourceName,
+				ImportState:      true,
+				ImportStateCheck: testAccNewRelicAlertPolicyImportStateCheckFunc(),
```

Comment:
Can this be replaced with a call to `TestCheckResourceAttr`?

Reply:
`ImportStateCheck` expects the type `ImportStateCheckFunc` which works with different types in the arguments that are passed to the pointer function.